pages/login.js

```diff
@@ -0,0 +1,61 @@
+import Head from 'next/head';
+import { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Login() {
+  const [error, setError] = useState('');
+
+  function onSubmit(event) {
+    event.preventDefault()
+
+    // Get form data and create JSON object
+    const formData = new FormData(event.target)
+    const formDataObj = {}
+    formData.forEach((value, key) => (formDataObj[key] = value))
+
+    let username = formDataObj["username"]
+    let password = formDataObj["password"]
+
+    // Check if user input username and password
+    if (username.length == 0 || password.length == 0) {
+      setError("Please enter both a Username and Password.")
+      return
+    }
+    else {
+      setError("")
+    }
```

Comment:
nit: `else` blocks should begin on the same line as the end curly brace of an `if` block
```suggestion
    } else {
      setError("")
    }
```